packages/api-headless-cms-tasks/src/tasks/entries/useCases/EmptyTrashBinByModel/CreateDeleteEntriesTasks.ts

```diff
@@ -0,0 +1,96 @@
+import { ITaskResponseResult } from "@webiny/tasks";
+import { CmsEntryListParams } from "@webiny/api-headless-cms/types";
+import { EntriesTask, IDeleteTrashBinEntriesInput, IEmptyTrashBinByModelTaskParams } from "~/types";
+
+const DELETE_ENTRIES_IN_BATCH = 50;
+const DELETE_ENTRIES_WAIT_TIME = 5;
+
+export class CreateDeleteEntriesTasks {
```

Comment:
Since these classes are all "use cases", I would recommend creating one generic interface, and use it everywhere, in all use cases, something like this:

```ts
interface IUseCase<TInput, TOutput> {
    execute(params: TInput) : Promise<TOutput>
}

class CreateDeleteEntriesTasks implements IUseCase<IEmptyTrashBinByModelTaskParams, ITaskResponseResult> {...}
```